### ISSUE TITLE
Non-latin text language not showing in mac os

### ISSUE THREAD

**Issue by ecolortest** [JGRennison/OpenTTD-patches#183]
## Version of OpenTTD
jgrpp-0.35.1-H3be17758
## Expected result
should show the text
## Actual result
![Screen Shot 2020-08-20 at 21 32 29](https://user-images.githubusercontent.com/34646054/90795328-df327d80-e348-11ea-8cc0-c6e7c200de0e.png)

## Steps to reproduce
change to any non-latin language (korean, chinese, japanese, and more...)

**Comment by James103** [comment]
The relevant OpenTTD.cfg lines are:
```
small_font = [default: sprite]
medium_font = [default: sprite]
large_font = [default: sprite]
mono_font = [default: sprite]
```

**Comment by JGRennison** [comment]
It may be worth trying to compile and test the upstream master branch (i.e. recent vanilla, not the release branch).
There have been various changes to the build system and OSX-related functionality since the 1.10 release branch.
There isn't all that much that I can do about OSX-specific issues.

**Comment by JGRennison** [comment]
Fixed in v0.37.0
